e2e/vite-integration/jest.config.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Please make this file to become `jest.config.ts` and use proper `defineConfig` function from `jest`